dom/events/scrolling/wheel-event-transactions-basic.tentative.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1">
```

Comment:
Please always add `<meta charset="utf-8">` to skip maybe expensive auto-detection of the browsers.

And shouldn't this test have `<meta name="timeout" content="long">` for avoiding unexpected timeout during the test?